src/privileged/FirefoxMonitor.jsm

```diff
@@ -156,9 +161,9 @@ this.FirefoxMonitor = {
     // accepts. moz-extension: is not one of them, so we work around that
     // by reading the file manually and creating a data: URL (allowed).
     // TODO:
-    // - Check locale and load relevant file
     // - Optimize?
-    let response = await fetch(this.getURL("locales/en_US/strings.properties"));
+    let locale = Services.locale.defaultLocale;
+    let response = await fetch(this.getURL(`locales/${locale}/strings.properties`));
```

Comment:
Is that guaranteed to exist? Should you catch and default to en_US?

Reply:
https://searchfox.org/mozilla-central/rev/50ba1dd30cf013bddce1ae756f1b3c95b26f0628/intl/locale/mozILocaleService.idl#55

^ suggests that it should always exist

Reply:
Oh hmm, just understood what you meant.

Reply:
Added a fallback mechanism if we don't have strings for defaultLocale. Probably an unlikely scenario since we are controlling distribution, but good catch! Thanks.